JavaFiddle-Web/web/resources/js/modal.js

```diff
@@ -788,19 +788,21 @@ function getPackagesList(projectname) {
 
 function isRightProjectName(name) {
```

Comment:
It should be marked as "TODO". Becouse  neither on server side nor on the client side we don't have any validation methods. Shall I add this mark explicitly?